plugins/colorschemes/dracula-nvim.nix

```
@@ -0,0 +1,20 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+lib.nixvim.neovim-plugin.mkNeovimPlugin config {
```

This recently changed, 

```suggestion
lib.nixvim.neovim-plugin.mkNeovimPlugin {
```